package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tg-wakatime-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/mohammadizanloo55/tg-wakatime-bot.git",
+  "author": "mohammadizanloo55 <mohammadizanloo55@gmail.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@fullstacksjs/eslint-config": "^6.5.2",
+    "eslint": "^7.32.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
```

Comment:
This is not a good idea.

Reply:
All extensions prettier can support
I don't understand the drawbacks of this

Reply:
Why? What is the problem with this?